application/microfrontends/eh-app/public/index.html

```diff
@@ -61,8 +61,10 @@
             .success(onInit);
 
         }
-        return initKeycloak;
-      })();
+
```

Comment:
I've seen that when the page is refreshed from the browser the configurations inject fails, it doesn't happen when you click on the Home button.
It seems that injectConfigIntoMfe was not satisfied.
I ran it on okd and locally and it seems ok.
please, check if frontend side is ok